tests/framework/e2e/downgrade.go

```diff
@@ -82,48 +81,55 @@ func DowngradeCancel(t *testing.T, epc *EtcdProcessCluster) {
 	t.Log("Cluster downgrade cancellation is completed")
 }
 
-func DowngradeUpgradeMembers(t *testing.T, lg *zap.Logger, clus *EtcdProcessCluster, numberOfMembersToChange int, currentVersion, targetVersion *semver.Version) error {
+func DowngradeUpgradeMembers(t *testing.T, lg *zap.Logger, clus *EtcdProcessCluster, numberOfMembersToChange int, newExecPath string, targetVersion version.Versions) error {
```

Comment:
Do we still need to update the signature of this function after introducing the downgrade query API?

Reply:
I was thinking it's better to pass expected version by caller instead of inferring by number of members.
Let me think over it and update it later.